src/app/vendors/Vendors.tsx

```diff
@@ -323,8 +317,7 @@ function VendorGroup({
   ownedItemHashes,
   currencyLookups,
   defs,
-  filtering,
-  vendorEngramDrops
```

Comment:
This is still present in the props definition for this component, and you probably still want it to pass into `Vendor`.